mempool/bench_test.go

```diff
@@ -2,23 +2,107 @@ package mempool
 
 import (
 	"encoding/binary"
+	"fmt"
+	"strconv"
+	"strings"
 	"testing"
 
-	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/abcix/example/kvstore"
 	cfg "github.com/tendermint/tendermint/config"
 	"github.com/tendermint/tendermint/proxy"
+	"github.com/tendermint/tendermint/types"
 )
 
-func BenchmarkCheckTx(b *testing.B) {
+var txs types.Txs
+
+//goos: darwin
+//goarch: amd64
+//pkg: github.com/tendermint/tendermint/mempool
+//BenchmarkClistCheckTx-8                             3585            318668 ns/op
+//BenchmarkLLRBCheckTx-8                              3415            320087 ns/op
+//BenchmarkBTreeCheckTx-8                             3588            306634 ns/op
+//BenchmarkClistRemoveTx-8                          812334              1522 ns/op
+//BenchmarkLLRBRemoveTx-8                           803126              1466 ns/op
+//BenchmarkBTreeRemoveTx-8                          823885              1445 ns/op
+//BenchmarkCacheInsertTime-8                       1690495               671 ns/op
+//BenchmarkCacheRemoveTime-8                       2572720               421 ns/op
+//BenchmarkClistMempoolGetNextTxBytes-8                  3         429128491 ns/op
+//BenchmarkLLRBMempoolGetNextTxBytes-8                 222           5250141 ns/op
+//BenchmarkBTreeMempoolGetNextTxBytes-8                142           9214519 ns/op
+
+func initTxs(size int) {
```

Comment:
just do it in `init`, then you don't need to call it every time https://golang.org/doc/effective_go.html#init